pkg/asset/installconfig/kubevirt/validation.go

```diff
@@ -16,6 +16,7 @@ func Validate(ic *types.InstallConfig, client Client) error {
 
 	allErrs = append(allErrs, validateStorageClassExistsInInfraCluster(kubevirtPlatform.StorageClass, client, fldPath.Child("storageClass"))...)
 	allErrs = append(allErrs, validateNetworkAttachmentDefinitionExistsInNamespace(kubevirtPlatform.NetworkName, kubevirtPlatform.Namespace, client, fldPath.Child("networkName"))...)
+	allErrs = append(allErrs, validateKubeMacPoolComponentEnabledForNamespace(kubevirtPlatform.Namespace, client, fldPath.Child("KubeMacPoolComponent "))...)
```

Comment:
There is a space in the string "KubeMacPoolComponent "
Are you sure it's working for you?

Reply:
Yes it worked (its only string), will remove